### DISCUSSION TITLE
Modern Search V4: Refinement filter for Yes/No column (RefinableYesNo*) not working

### DISCUSSION THREAD

**Discussion by Sigi70** [microsoft-search/pnp-modern-search#2326] [Q&A] [answered]
Hello Everyone,

I tried to filter results by managed property RefinableYesNo00 using 
`RefinableYesNo:equals("true")`

Debugging view shows correct data content
either 
`"RefinableYesNo00": "true",` or
`"RefinableYesNo00": "false",`

but result is 0.

Wondering if my filter term is not correct, it's a bug or missing feature?

Regards,
Sigi

**Comment by wobba** [comment, marked as answer]
I would try it this way instead:

`RefinableYesNo00:1`
or
`RefinableYesNo00:true`

`equals` is meant for strings, not boolean values.

**Reply by Sigi70** [reply]
Many thx 
`RefinableYesNo00:true` is the way :-)

`RefinableYesNo00:1` didn't work.

**Reply by wobba** [reply]
I knew one of them would work 😁
